uwb/example/lib/host_widgets/distance.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+/// The [Distance] class that shows the distance of the connected phone.
+class Distance extends StatelessWidget {
+  /// The [Distance] constructor.
+  const Distance(
+      {Key? key, required this.x, required this.y, required this.distance})
+      : super(key: key);
+
+  /// Variable for x-axis of the distance widget.
+  final double x;
+
+  /// Variable for y-axis of the distance widget.
+  final double y;
+
+  /// Variable that contains the distance to the other device.
```

Comment:
A better description would be (of course you should replace the `xx` with the unit of measurement, e.g. centimeters):

```suggestion
  /// The distance in xx to the other device.
```